remoteit/desktop
# feature(): dark mode integrate material ui

### Description

feature(): dark mode integrate material ui


### Test plan

1. Sign in
2. Go to sistem preference
3. Change to dark mode and you should see the new UI

### Ticket
https://remoteit.atlassian.net/browse/DESK-1274
